packages/styles/framework/src/utilities/mixins/_mixins.scss

```
@@ -155,20 +210,58 @@
   .#{$type}-tile {
     position: relative;
-    max-width: 160px;
-    display: inline-block;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    padding: 40px 16px 16px;
+    border: var(--border);
+    border-radius: var(--radius);
+    box-sizing: border-box;
+    background-color: getColor('background');
```

Stylelint (17.3.0)

[error] Expected "getColor" to be "getcolor" (function-name-case)

(function-name-case)